src/core_modules/capture-core/components/Pages/EditEvent/DataEntry/EditEventDataEntry.component.js

```diff
@@ -319,7 +319,7 @@ type DataEntrySection = {
     name: string,
 };
 
-const dataEntrySectionDefinitions = {
+const dataEntrySectionDefinitions = () => ({
```

Comment:
It seems like we are not invoking this function? (and some of the other functions in this PR).

Reply:
I added the missing invoking. Thanks for the feedback!